react/OrderPayment.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { createContext, ReactNode, useContext } from 'react'
+import { useQuery, useMutation } from 'react-apollo'
+import { cardSessionId as CardSessionIdQuery } from 'vtex.checkout-resources/Queries'
+import {
+  savePaymentToken as SavePaymentTokenMutation,
+  saveCards as SaveCardsMutation,
+} from 'vtex.checkout-resources/Mutations'
```

Comment:
```suggestion
import SavePaymentTokenMutation from 'vtex.checkout-resources/MutationSavePaymentToken'
import SaveCardsMutation from 'vtex.checkout-resources/MutationSaveCards'
```
Context: https://github.com/vtex-apps/checkout-resources/pull/22

Reply:
I think it is a major change.

Reply:
For what reason? These are not breaking changes.

Reply:
Ohh sorry, the other way still works.